### DISCUSSION TITLE
VCS tags missing

### DISCUSSION THREAD

**Discussion by nessdoor** [Wimmics/corese#100] [Q&A] [answered]
Hello,

I am trying to build a reasonable container image for Corese, compiling directly from source. I noticed that you stopped tagging releases on Git a while ago. Is there any reason behind this? Can you resume tagging releases?

i don't want to directly track master, since new pushes might break my container builds. Neither I do want to rely on your binary distributions.

The painful workaround, for now, requires reading through the recent commit history to checkout specific commits with stable code.

**Comment by remiceres** [comment, marked as answer]
Hello,

Thanks for your message, I understand that it is difficult to build a container image for Corese if the releases are not tagged.
Good news, we are resuming tagging releases.

The latest version at this date is 4.3.0 [available here](https://github.com/Wimmics/corese/releases/tag/release-4.3.0). Future versions will be tagged in the same way.

Also, we are publishing an official container image for Corese-Server available on [Docker Hub](https://hub.docker.com/r/wimmics/corese).

**Reply by nessdoor** [reply]
Very good to hear all of this, thank you for your efforts.
Looking forward to the next releases, too.
